### PR TITLE
Fix: Lua directory empty and always loading in editor

### PR DESCRIPTION
**Pull request by tjmgd**
Signed-off-by: T.J. McGrath-Daly <tj.mcgrath.daly@huawei.com>

### CODE REVIEW

**Review by lsemp3d — APPROVED** [review]

File: Code/Tools/Standalone/Source/LUA/LUAEditorMainWindow.cpp [hunk 1]
```diff
@@ -346,7 +346,11 @@ namespace LUAEditor
             {
                 auto selectedAsset = selectedAssets.front();
                 const AZStd::string filePath = selectedAsset->GetFullPath();
-                EBUS_EVENT(Context_DocumentManagement::Bus, OnLoadDocument, filePath, true);
+                auto entry_type = selectedAsset->GetEntryType();
+                if (entry_type == AzToolsFramework::AssetBrowser::AssetBrowserEntry::AssetEntryType::Source)
```
Comment (lumberyard-employee-dm) [inline Code/Tools/Standalone/Source/LUA/LUAEditorMainWindow.cpp:350]: standards: The variable naming scheme should be [lowerCamelCase](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#variable-names) according to the Coding Standards

Comment (tjmgd) [reply Code/Tools/Standalone/Source/LUA/LUAEditorMainWindow.cpp]: Apologies. This has now been fixed.

**Review by amznestebanpapp — APPROVED** [review]

**Comment by amznestebanpapp** [comment]
@tjmgd can you fix the current conflicts? I can trigger the validation build after the conflicts are resolved. Thanks

**Comment by tjmgd** [comment]
Updated now

**Pull request merged**